plasma_framework/contracts/poc/fast_exits/Liquidity.sol

```diff
@@ -153,23 +148,49 @@ contract Liquidity {
         return true;
     }
 
+    /**
+     * @notice mint an ERC-721 wrapping the exit
+     * @param rlpOutputTxToContract RLP-encoded transaction that creates the outputs for the contract
+     * @param utxoPosToExit position of the output which the contract has to exit
+     * @param decodedSecondTx decoded second transaction
+    */
+    function mintNFT(bytes memory rlpOutputTxToContract, uint256 utxoPosToExit, PaymentTransactionModel.Transaction memory decodedSecondTx) private {
+        //change the return type of exitId once the pr to change it has been merged
+        uint160 exitId = paymentExitGame.getStandardExitId(false, rlpOutputTxToContract, utxoPosToExit);
+        super._mint(msg.sender, exitId);
+
+        FungibleTokenOutputModel.Output memory outputFromSecondTransaction
+        = decodedSecondTx.outputs[0];
+        exitData[exitId] = ExitData(msg.value, outputFromSecondTransaction.amount);
+    }
+
+    /**
+     * @dev Check if the token for the exit exists
+     * @param tokenId The tokenId which is also the exitId
+    */
+    function exists(uint256 tokenId) public view returns (bool) {
+        return _exists(tokenId);
+    }
```

Comment:
why are we wrapping this method? We should probably stick to ERC721 interface as much as possible.

Reply:
so `_exists` from the ERC721 is internal. We could contrarily also check with `owner == address(0)` but this improves readability?

Reply:
where do we use it? only in tests? we should try to avoid writing code that is used only in tests.

Reply:
Aah yess, my bad :man_facepalming: . removed `9651898`